### PR TITLE
poppler: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by maxim-belkin**

### CODE REVIEW

**Comment by maxim-belkin** [comment]
situation "brown":
```
checking for CAIRO... no
configure: error: "Cairo output is required to build glib frontend"
```
at the same time
```
depends_on "cairo"
```
is in the formula... `libpthreads-stubs` again?

**Comment by maxim-belkin** [comment]
confirmed locally: adding
```
depends_on "linuxbrew/xorg/libpthread-stubs" => :build unless OS.mac?
```
solves the issue
@sjackman thoughts?
@rwhogg your idea to implement `libpthread-stubs` as a formula instead of a patch in `xorg` is paying off! 👍

File: Formula/poppler.rb [hunk 1]
```diff
@@ -33,7 +34,10 @@ class Poppler < Formula
   depends_on "little-cms2" => :optional
 
   # workaround for incorrect X11 dependency propogation by cairo
-  depends_on :x11 unless OS.mac?
+  unless OS.mac?
+    depends_on :x11
+    depends_on "linuxbrew/xorg/libpthread-stubs" => :buil
```
Comment (sjackman) [inline Formula/poppler.rb]: Typo: `:buil` should be `:build`.

**Comment by sjackman** [comment]
I just looked in `libpthread-stubs`, and there's nothing in it! Just an essentially empty `pkgconfig/pthread-stubs.pc` file! Should we make `:x11` depend on `linuxbrew/xorg/libpthread-stubs`?

**Pull request closed without merging**